src/main/java/edu/harvard/iq/dataverse/api/Pids.java

```diff
@@ -48,8 +48,12 @@ public Response getPid(@QueryParam("persistentId") String persistentId) {
         String baseUrl = systemConfig.getDataCiteRestApiUrlString();
         String username = System.getProperty("doi.username");
         String password = System.getProperty("doi.password");
-        JsonObjectBuilder result = PidUtil.queryDoi(persistentId, baseUrl, username, password);
-        return ok(result);
+        try {
+            JsonObjectBuilder result = PidUtil.queryDoi(persistentId, baseUrl, username, password);
+            return ok(result);
+        } catch (Exception ex) {
+            return error(Response.Status.BAD_REQUEST, ex.getLocalizedMessage());
```

Comment:
Since this is a config problem, BAD_REQUEST is probably not the right response.

Reply:
@qqmyers yeah, 500 probably makes more sense. Fixed in 7c55e2bd3.